### PR TITLE
Split internal non-stable publishing feed config

### PR DESCRIPTION
**Pull request by JohnTortugo**
Relates to this comment: https://github.com/dotnet/arcade/pull/3882#discussion_r321817947
Relates to this issue: #3868 
Relates to this issue: https://github.com/dotnet/arcade/issues/3890

Split internal non-stable publishing into separate feed configs: non shipping and shipping feeds

### CODE REVIEW

**Comment by JohnTortugo** [comment]
This add more clean up that I'll do as part of [this issue here](https://github.com/dotnet/arcade/issues/3842)

File: src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/SetupTargetFeeds.proj [hunk 1]
```diff
@@ -40,7 +40,8 @@
       - AzureDevOpsFeedsBaseUrl
       - ArtifactsCategory                     : Used to let user override target feed of public channels.
       - AzureStorageTargetFeedPAT
-      - StaticInternalFeed                    : URL of the feed to publish internal non-stable packages
+      - StaticInternalShippingFeed            : URL of the feed to publish shipping internal non-stable packages
```
Comment (riarenas) [inline src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/SetupTargetFeeds.proj]: These two new properties are not being passed by anyone. Am I missing where they are being added?

Comment (JohnTortugo) [reply src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/SetupTargetFeeds.proj]: I need to get a new version of the SDK first.

Comment (riarenas) [reply src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/SetupTargetFeeds.proj]: Ahh, I guess I got confused due to thinking this PR would be enough to close https://github.com/dotnet/arcade/issues/3890, but there should be another PR where the parameter is passed to the channel yamls?

**Comment by mmitche** [comment]
Closing in favor of #3929

**Pull request closed without merging**